src/main/java/hudson/plugins/clover/CloverPublisher.java

```diff
@@ -123,12 +127,21 @@ public void setFailingTarget(CoverageTarget failingTarget) {
     /**
      * Gets the directory where the Clover Report is stored for the given build.
      */
+    @Deprecated
```

Comment:
I don't know if it has sense , but when I see a `Deprecated` annotation I like to see the alternative so I don't have to navigate through all class searching the new method.